Assets/Scripts/StandardSamples/UI/PlayerReports/UIPlayerReportMenu.cs

```diff
@@ -49,7 +49,7 @@ public class UIPlayerReportMenu : SampleMenuWithFriends
         private EOSFriendsManager FriendsManager;
         
 
-        private void Start()
+        new private void Awake()
         {
```

Comment:
You very rarely want to use the `new` function modifier. By doing this you are stopping the function in the base class from being called
```suggestion
        protected override void Awake()
        {
            base.Awake();
```